setup.py

```diff
@@ -20,7 +20,7 @@
         "numpy>=1.11, <2.0",
         "pyyaml>=4.2b1, <5.0",
         "scipy>=1.1.0, <2.0.0",
-        "snorkel-metal>=0.4.1, <0.5.0",
+        "snorkel>=0.9.0, <0.10.0",
```

Comment:
This is not the latest version of snorkel. The latest version fixed the networkx version.

Reply:
Resolved by b85300a10fbc2e21220d68953c245f0d67d08f90.